Makefile

```diff
@@ -33,6 +33,14 @@ tropo-lint:
 node-test: ./lambda-src/node_modules/
 	cd lambda-src && npm test
 
+node-lint: ./lambda-src/node_modules/
+	./lambda-src/node_modules/.bin/standard lambda-src/*.js
+	./lambda-src/node_modules/.bin/standard --env mocha test/*.js
+
+node-lint-fix: ./lambda-src/node_modules/
+	./lambda-src/node_modules/.bin/standard --fix lambda-src/*.js
+	./lambda-src/node_modules/.bin/standard --fix --env mocha test/*.js
```

Comment:
Would be a bit nicer to have an npm script for running this. (As it sorts out the path and is a bit more idiomatic.)

e.g. `npm run lint`

That would also give us the option of side-stepping make in CI if that's more convenient.

Reply:
#9

Reply:
PR accepted. Thanks @daniel-ac-martin